astro/src/content/docs/apis/_user-request-body.mdx

```diff
@@ -126,6 +131,10 @@ You must specify either the **email** or the **username** or both for the User.
 
   <APIField name="user.mobilePhone" type="String" optional>
     The User's mobile phone number. This is useful is you will be sending push notifications or SMS messages to the User.
+
+    <Aside>
+      This differs from <InlineField>user.phoneNumber</InlineField> and cannot be used to log in.
```

Comment:
Might be clearer to name each with its use in the aside.

```suggestion
<InlineField>user.mobilePhone</InlineField> is a user attribute that cannot be used to log in. For logging in with a phone number, see <InlineField>user.phoneNumber</InlineField>.```

Reply:
Fixed